src/Microsoft.AspNetCore.SignalR.Core/HubEndPoint.cs

```diff
@@ -275,6 +275,20 @@ public class HubEndPoint<THub> : IInvocationBinder where THub : Hub
                                         _ = ProcessInvocation(connection, invocationMessage);
                                         break;
 
+                                    case CancelInvocationMessage cancelInvocationMessage:
+                                        //log
+                                        // check and cancel stream
+                                        if (connection.ActiveRequests.TryRemove(cancelInvocationMessage.InvocationId, out var cts))
+                                        {
+                                            cts.Cancel();
+                                        }
+                                        else
+                                        {
+                                            // stream can be canceled on the server while client is canceling stream
+                                            _logger.LogDebug("CancelInvocationMessage received unexpectedly.");
```

Comment:
👍

Reply:
Make this message part of the other logging messages